content/docs/1.4.0/deploy/uninstall/_index.md

```diff
@@ -13,6 +13,11 @@ In this section, you'll learn how to uninstall Longhorn.
 - [Troubleshooting](#troubleshooting)
 
 ### Prerequisite
+To prevent Longhorn from being accidentally uninstalled (which leads to data lost),
+we introduce a new setting, [deleting-confirmation-flag](../../references/settings/#deleting-confirmation-flag).
+If this flag **false**, Longhorn uninstallation job will fail.
+Set this flag to **true** to allow Longhorn uninstallation.
```

Comment:
NIT: Mentioning how to modify this. I mean, providing cmd `kubectl -n longhorn-system patch -p '{"value": "true"}' --type=merge lhs deleting-confirmation-flag`

Reply:
Thanks. Updated